editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -2600,6 +2623,21 @@ void SpatialEditorViewport::_menu_option(int p_option) {
 			current = !current;
 			view_menu->get_popup()->set_item_checked(idx, current);
 		} break;
+		case VIEW_CINEMATIC_PREVIEW: {
+			int idx = view_menu->get_popup()->get_item_index(VIEW_CINEMATIC_PREVIEW);
+			bool current = view_menu->get_popup()->is_item_checked(idx);
```

Comment:
I'd rename this bool to `activate` or `enable`, `current` is misleading as it's also used to qualify the "current (active) camera".

Reply:
Ah well I see it was already done this way for other tools above, so it's fine.

That's a weird way to handle toggles though, I'm pretty sure there should be a cleaner way to handle this.